app/controllers/concerns/oregon_digital/blacklight_config_behavior.rb

```diff
@@ -70,7 +70,7 @@ def self.modified_field
         #   The ordering of the field names is the order of the display
         config.add_index_field 'title_tesim', label: 'Title', itemprop: 'name', if: false, highlight: true
         config.add_index_field 'creator_label_sim', itemprop: 'creator', link_to_search: 'creator_label_sim', max_values: 3, max_values_label: 'others'
-        config.add_index_field 'date_tesim', itemprop: 'date'
+        config.add_index_field 'date_tesim'
```

Comment:
We need the `itemprop` for the work show page.
Can you add `itemscope itemtype="http://schema.org/CreativeWork"` to here: https://github.com/OregonDigital/OD2/blob/master/app/views/catalog/_index_gallery.html.erb#L1

That should turn the result into a proper item